packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3232,70 +3243,94 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
     bool dropSemanticsOfPreviousSiblings = config.isBlockingSemanticsOfPreviouslyPaintedNodes;
 
     final bool producesForkingFragment = !config.hasBeenAnnotated && !config.isSemanticBoundary;
-    final List<_InterestingSemanticsFragment> fragments = <_InterestingSemanticsFragment>[];
-    final Set<_InterestingSemanticsFragment> toBeMarkedExplicit = <_InterestingSemanticsFragment>{};
     final bool childrenMergeIntoParent = mergeIntoParent || config.isMergingSemanticsOfDescendants;
-
+    final List<SemanticsConfiguration> childConfigurations = <SemanticsConfiguration>[];
+    final bool explicitChildNode = config.explicitChildNodes || parent is! RenderObject;
+    final bool hasSemanticsMerger = config.childConfigsDelegate != null;
```

Comment:
same here about the word "merger". Can you audit the entire change to make sure we eliminate this name since the concept no longer exists?